ls1intum/artemis-android
# `Communication`: Add Post Action Bar to Thread View

### Problem Description

The Post Actions are not as accessible through the bottom sheet dialog. The iOS app has a post action bar placed below the parent post in the thread view.

### Changes

This PR introduces the post action bar for Android, allowing users to easily access important post actions. 
The post action bar has been placed between the standalone post and the answer posts in MetisThreadUI. Post colors have been moved to a common object in the ui package.

### Steps for testing

1. Go to a chat.
2. Click on a post to navigate to the thread UI.
3. Verify that the answer post count is correct.
4. Verify that the following post actions are available:
    - deleting for authors and moderators
    - editing for authors
    - pining for moderators in channels, for group creators in groups
    - copying for everyone

### Screenshots

![postActionBar (1)](https://github.com/user-attachments/assets/cccf4744-ed9a-415c-8b5e-a99a6bea7315)
